notebooks/hello_world_thunderfx.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>Instead of promising "significant speedups," let's tweak the second sentence of the first paragraph to something like:</p><p> </p><p>"... optimized kernels as it executes. Compiling these kernels can take seconds or even minutes for larger models, but each kernel only has to be compiled once."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>